udata_recommendations/views.py

```diff
@@ -8,14 +8,30 @@
 blueprint = Blueprint('recommendations', __name__, template_folder='templates')
 
 
+NB_RECOMMENDATIONS = 2
+
+
 def has_recommendations(ctx):
     dataset = ctx['dataset']
-    return 'recommendations' in dataset.extras and len(dataset.extras['recommendations'])
+    return dataset and len(dataset.extras.get('recommendations', []))
```

Comment:
maybe we should protect against an existing key but with `None` value (this is editable by producers via the API)

Reply:
Maybe even against anything that's not a list.